torchvision/prototype/models/googlenet.py

```diff
@@ -17,7 +17,7 @@
 
 
 class GoogLeNetWeights(Weights):
-    ImageNet1K_TheCodezV1 = WeightEntry(
+    ImageNet1K_Community = WeightEntry(
```

Comment:
Changed the enum convention for weights that come from the community and are not part of another library.